components/StoryCards/StoryCard.module.scss

```diff
@@ -27,7 +22,7 @@ $spacing-card: $spacing-4 - $spacing-half; // 1.25rem = 20px That's the only hac
     display: block;
     overflow: hidden;
     text-decoration: none;
-    border: 1px solid $color-borders;
+    border: 1px solid var(--prezly-border-color);
```

Comment:
a touch too strong? should we drop the opacity of this border a little?
![CleanShot 2024-08-19 at 12 42 39@2x](https://github.com/user-attachments/assets/8b817abd-f31d-4be8-b0d3-600516b393d6)

Reply:
I can barely notice the border to be honest.